polaris/ocean/tasks/manufactured_solution/manufactured_solution.cfg

```diff
@@ -30,8 +30,8 @@ dt_per_km = 3.0
 # Convergence threshold below which the test fails
 conv_thresh = 1.8
 
-# Convergence rate above which a warning is issued
-conv_max = 2.2
+# Run duration in days
+run_duration = 0.4166667
```

Comment:
This suggests that a run duration in days isn't great.  Should we make it hours? Or seconds?

Reply:
Hours seems reasonable to me. I'll add that change as well.

Reply:
Great, yeah, I think that's best.